src/Elm/Processing.elm

```diff
@@ -67,118 +63,7 @@ addDependency dep (ProcessContext x) =
-            , remainingComments = context.remainingComments
-            , declarations = declaration :: context.declarations
-            }
-
-        InfixDeclaration _ ->
-            { previousComments = context.previousComments
-            , remainingComments = context.remainingComments
-            , declarations = declaration :: context.declarations
-            }
-
-        Destructuring _ _ ->
-            -- Will never happen. Will be removed in v8
-            { previousComments = context.previousComments
-            , remainingComments = context.remainingComments
-            , declarations = declaration :: context.declarations
-            }
-
-
-addDocumentation : (Node Comment -> Declaration) -> Node Declaration -> DeclarationsAndComments -> DeclarationsAndComments
-addDocumentation howToUpdate declaration file =
-    let
-        ( previous, maybeDoc, remaining ) =
-            findDocumentationForRange (Node.range declaration) file.remainingComments []
-    in
-    case maybeDoc of
-        Just doc ->
-            { previousComments = previous ++ file.previousComments
-            , remainingComments = remaining
-            , declarations = Node { start = (Node.range doc).start, end = (Node.range declaration).end } (howToUpdate doc) :: file.declarations
-            }
-
-        Nothing ->
-            { previousComments = previous ++ file.previousComments
-            , remainingComments = remaining
-            , declarations = declaration :: file.declarations
-            }
-
-
-findDocumentationForRange : Range -> List (Node String) -> List (Node String) -> ( List (Node String), Maybe (Node String), List (Node String) )
-findDocumentationForRange range comments previousComments =
-    case comments of
-        [] ->
-            ( previousComments, Nothing, [] )
-
-        ((Node commentRange commentText) as comment) :: restOfComments ->
-            -- Since both comments and declarations are in the order that they appear in the source code,
-            -- all the comments we've evaluated until now don't need to be re-evaluated when
-            -- trying the find the documentation for later declarations if the current comment is later than the current declaration.
-            case compare (commentRange.end.row + 1) range.start.row of
-                EQ ->
-                    if String.startsWith "{-|" commentText then
-                        ( previousComments, Just comment, restOfComments )
-
-                    else
-                        -- Aborting because the next comment can't match the next declaration
-                        ( previousComments, Nothing, comment :: restOfComments )
-
-                LT ->
-                    findDocumentationForRange range restOfComments (comment :: previousComments)
-
-                GT ->
-                    -- Aborting because we went too far
-                    ( previousComments, Nothing, comment :: restOfComments )
+    file
```

Comment:
This function is a bit awkward now, it basically does nothing :)

Reply:
And that's great :grin: 
I'll deprecate it at some point, and it will be removed at the next major version :muscle: